halo2_proofs/tests/plonk_api.rs

```diff
@@ -643,532 +643,6 @@ fn plonk_api() {
         );
     }
 
-    fn test_plonk_api_ipa() {
-        use halo2_proofs::poly::ipa::commitment::{IPACommitmentScheme, ParamsIPA};
-        use halo2_proofs::poly::ipa::multiopen::{ProverIPA, VerifierIPA};
-        use halo2_proofs::poly::ipa::strategy::AccumulatorStrategy;
-        use halo2curves::pasta::EqAffine;
-
-        type Scheme = IPACommitmentScheme<EqAffine>;
-        bad_keys!(Scheme);
-
-        let mut rng = test_rng();
-        let params = ParamsIPA::<EqAffine>::new(K);
-
-        let pk = keygen::<IPACommitmentScheme<EqAffine>>(&params);
-
-        let proof = create_proof::<_, ProverIPA<_>, _, _, Blake2bWrite<_, _, Challenge255<_>>>(
-            &mut rng, &params, &pk,
-        );
-
-        let verifier_params = params;
-
-        verify_proof::<
-            _,
-            VerifierIPA<_>,
-            _,
-            Blake2bRead<_, _, Challenge255<_>>,
-            AccumulatorStrategy<_>,
-        >(&verifier_params, pk.get_vk(), &proof[..]);
-
-        // Check that the verification key has not changed unexpectedly
-        {
-            // panic!("{:#?}", pk.get_vk().pinned());
-            assert_eq!(
-                format!("{:#?}", pk.get_vk().pinned()),
-                r#"PinnedVerificationKey {
-    base_modulus: "0x40000000000000000000000000000000224698fc0994a8dd8c46eb2100000001",
```

Comment:
I think we need to keep this test - `PinnedVerificationKey` comparison, somewhere.
Please add test in kzg tests.

Reply:
Hmm, it is also good.
What I wanted was to add the test comparing the debug representation of `PinnedVerificationKey` with real one.
If we see the change in representation, it would be more clear that we should update the proof hash(since it's changed)